metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/DirectSqlTable.java

```diff
@@ -222,6 +228,54 @@ public Long getTableId(final QualifiedName tableName) {
         }
     }
 
+    /**
+     * Deletes all the table related information from the store.
+     * @param tableName table name
+     */
+    public void delete(final QualifiedName tableName) {
+        try {
+            final TableSequenceIds ids = getSequenceIds(tableName);
+            directSqlSavePartition.delete(tableName);
+            jdbcTemplate.update(SQL.UPDATE_SDS_CD, new SqlParameterValue(Types.BIGINT, null),
+                new SqlParameterValue(Types.BIGINT, ids.getSdsId()));
```

Comment:
I am not sure if we want to set the value to null instead of deleting them. I guess that's hive requirement?

Reply:
This is how the foreign key relations on a table are designed.

Reply:
ok.